sql/driver/driver_test.go

```diff
@@ -350,6 +352,19 @@ func TestInsertSelectDelete(t *testing.T) {
 		if _, err := db.Exec(fmt.Sprintf(`INSERT INTO %s.kv VALUES ('e', 'f')`, testcase.db)); err != nil {
 			t.Fatal(err)
 		}
+		// Primary key uniqueness violation.
+		if _, err := db.Exec(fmt.Sprintf(`INSERT INTO %s.kv VALUES ('e', 'f')`, testcase.db)); !isError(err, "unexpected value") {
```

Comment:
done. left a TODO to figure out how to print the specific index that is being violated.